test/e2e-go/features/catchup/basicCatchup_test.go

```diff
@@ -75,6 +79,38 @@ func TestBasicCatchup(t *testing.T) {
 	// Now, catch up
 	err = fixture.LibGoalFixture.ClientWaitForRoundWithTimeout(cloneClient, waitForRound)
 	a.NoError(err)
+
+	var status model.NodeStatusResponse
+
+	cloneNC := fixture.GetNodeControllerForDataDir(cloneDataDir)
+	cloneRestClient := fixture.GetAlgodClientForController(cloneNC)
+
+	reportLog := func() {
+		status, err = cloneRestClient.Status()
+		a.NoError(err)
+		primStatus, err := fixture.GetAlgodClientForController(nc).Status()
+		a.NoError(err)
+		log.Infof("currend round: %d, primary round: %d, current-sync time and time since last round %f, %f, prim-sync time and time since last round: %f, %f, 2nd node catchpoint: %v",
+			status.LastRound, primStatus.LastRound,
+			time.Duration(status.CatchupTime).Seconds(), time.Duration(status.TimeSinceLastRound).Seconds(),
+			time.Duration(primStatus.CatchupTime).Seconds(), time.Duration(primStatus.TimeSinceLastRound).Seconds(),
+			*status.Catchpoint)
+	}
+
+	// an immediate call for ready will error, for sync time != 0
+	a.Error(cloneRestClient.ReadyCheck())
+
+	for {
+		reportLog()
```

Comment:
at least this test have a node sync time get back to const 0 when catching up, log following:
```
    testingLogger.go:38: time="2023-03-13T14:24:38.219359 -0400" level=info msg="currend round: 3, primary round: 3, current-sync time and time since last round 0.075313, 0.072977, prim-sync time and time since last round: 0.000000, 0.258024, 2nd node catchpoint: " file=basicCatchup_test.go function=github.com/algorand/go-algorand/test/e2e-go/features/catchup.TestBasicCatchup.func1 line=93
    testingLogger.go:38: time="2023-03-13T14:24:38.471693 -0400" level=info msg="currend round: 3, primary round: 3, current-sync time and time since last round 0.327502, 0.325164, prim-sync time and time since last round: 0.000000, 0.510355, 2nd node catchpoint: " file=basicCatchup_test.go function=github.com/algorand/go-algorand/test/e2e-go/features/catchup.TestBasicCatchup.func1 line=93
    testingLogger.go:38: time="2023-03-13T14:24:38.723631 -0400" level=info msg="currend round: 3, primary round: 3, current-sync time and time since last round 0.579436, 0.577096, prim-sync time and time since last round: 0.000000, 0.762278, 2nd node catchpoint: " file=basicCatchup_test.go function=github.com/algorand/go-algorand/test/e2e-go/features/catchup.TestBasicCatchup.func1 line=93
    testingLogger.go:38: time="2023-03-13T14:24:38.974789 -0400" level=info msg="currend round: 3, primary round: 3, current-sync time and time since last round 0.830724, 0.828381, prim-sync time and time since last round: 0.000000, 1.013497, 2nd node catchpoint: " file=basicCatchup_test.go function=github.com/algorand/go-algorand/test/e2e-go/features/catchup.TestBasicCatchup.func1 line=93
    testingLogger.go:38: time="2023-03-13T14:24:39.226309 -0400" level=info msg="currend round: 3, primary round: 3, current-sync time and time since last round 1.082204, 1.079858, prim-sync time and time since last round: 0.000000, 1.265008, 2nd node catchpoint: " file=basicCatchup_test.go function=github.com/algorand/go-algorand/test/e2e-go/features/catchup.TestBasicCatchup.func1 line=93
    testingLogger.go:38: time="2023-03-13T14:24:39.477646 -0400" level=info msg="currend round: 3, primary round: 3, current-sync time and time since last round 1.333409, 1.331061, prim-sync time and time since last round: 0.000000, 1.516220, 2nd node catchpoint: " file=basicCatchup_test.go function=github.com/algorand/go-algorand/test/e2e-go/features/catchup.TestBasicCatchup.func1 line=93
    testingLogger.go:38: time="2023-03-13T14:24:39.728731 -0400" level=info msg="currend round: 3, primary round: 3, current-sync time and time since last round 1.584635, 1.582284, prim-sync time and time since last round: 0.000000, 1.767414, 2nd node catchpoint: " file=basicCatchup_test.go function=github.com/algorand/go-algorand/test/e2e-go/features/catchup.TestBasicCatchup.func1 line=93
    testingLogger.go:38: time="2023-03-13T14:24:39.980562 -0400" level=info msg="currend round: 3, primary round: 3, current-sync time and time since last round 1.836431, 1.834077, prim-sync time and time since last round: 0.000000, 2.019234, 2nd node catchpoint: " file=basicCatchup_test.go function=github.com/algorand/go-algorand/test/e2e-go/features/catchup.TestBasicCatchup.func1 line=93
    testingLogger.go:38: time="2023-03-13T14:24:40.231904 -0400" level=info msg="currend round: 3, primary round: 3, current-sync time and time since last round 2.087740, 2.085384, prim-sync time and time since last round: 0.000000, 2.270553, 2nd node catchpoint: " file=basicCatchup_test.go function=github.com/algorand/go-algorand/test/e2e-go/features/catchup.TestBasicCatchup.func1 line=93
    testingLogger.go:38: time="2023-03-13T14:24:40.483949 -0400" level=info msg="currend round: 3, primary round: 3, current-sync time and time since last round 2.339775, 2.337416, prim-sync time and time since last round: 0.000000, 2.522616, 2nd node catchpoint: " file=basicCatchup_test.go function=github.com/algorand/go-algorand/test/e2e-go/features/catchup.TestBasicCatchup.func1 line=93
    testingLogger.go:38: time="2023-03-13T14:24:40.737725 -0400" level=info msg="currend round: 3, primary round: 3, current-sync time and time since last round 2.591853, 2.589491, prim-sync time and time since last round: 0.000000, 2.776103, 2nd node catchpoint: " file=basicCatchup_test.go function=github.com/algorand/go-algorand/test/e2e-go/features/catchup.TestBasicCatchup.func1 line=93
    testingLogger.go:38: time="2023-03-13T14:24:40.989936 -0400" level=info msg="currend round: 3, primary round: 3, current-sync time and time since last round 2.845692, 2.843327, prim-sync time and time since last round: 0.000000, 3.028575, 2nd node catchpoint: " file=basicCatchup_test.go function=github.com/algorand/go-algorand/test/e2e-go/features/catchup.TestBasicCatchup.func1 line=93
    testingLogger.go:38: time="2023-03-13T14:24:41.241865 -0400" level=info msg="currend round: 3, primary round: 3, current-sync time and time since last round 3.097759, 3.095392, prim-sync time and time since last round: 0.000000, 3.280533, 2nd node catchpoint: " file=basicCatchup_test.go function=github.com/algorand/go-algorand/test/e2e-go/features/catchup.TestBasicCatchup.func1 line=93
    testingLogger.go:38: time="2023-03-13T14:24:41.493083 -0400" level=info msg="currend round: 3, primary round: 4, current-sync time and time since last round 3.348962, 3.346592, prim-sync time and time since last round: 0.000000, 0.113762, 2nd node catchpoint: " file=basicCatchup_test.go function=github.com/algorand/go-algorand/test/e2e-go/features/catchup.TestBasicCatchup.func1 line=93
    testingLogger.go:38: time="2023-03-13T14:24:41.744230 -0400" level=info msg="currend round: 3, primary round: 4, current-sync time and time since last round 3.600133, 3.597761, prim-sync time and time since last round: 0.000000, 0.364884, 2nd node catchpoint: " file=basicCatchup_test.go function=github.com/algorand/go-algorand/test/e2e-go/features/catchup.TestBasicCatchup.func1 line=93
    testingLogger.go:38: time="2023-03-13T14:24:41.995566 -0400" level=info msg="currend round: 3, primary round: 4, current-sync time and time since last round 3.851476, 3.849101, prim-sync time and time since last round: 0.000000, 0.616231, 2nd node catchpoint: " file=basicCatchup_test.go function=github.com/algorand/go-algorand/test/e2e-go/features/catchup.TestBasicCatchup.func1 line=93
    testingLogger.go:38: time="2023-03-13T14:24:42.247663 -0400" level=info msg="currend round: 4, primary round: 4, current-sync time and time since last round 4.103537, 0.101226, prim-sync time and time since last round: 0.000000, 0.868319, 2nd node catchpoint: " file=basicCatchup_test.go function=github.com/algorand/go-algorand/test/e2e-go/features/catchup.TestBasicCatchup.func1 line=93
    testingLogger.go:38: time="2023-03-13T14:24:42.499193 -0400" level=info msg="currend round: 4, primary round: 4, current-sync time and time since last round 4.355084, 0.352771, prim-sync time and time since last round: 0.000000, 1.119872, 2nd node catchpoint: " file=basicCatchup_test.go function=github.com/algorand/go-algorand/test/e2e-go/features/catchup.TestBasicCatchup.func1 line=93
    testingLogger.go:38: time="2023-03-13T14:24:42.750502 -0400" level=info msg="currend round: 4, primary round: 4, current-sync time and time since last round 4.606398, 0.604082, prim-sync time and time since last round: 0.000000, 1.371173, 2nd node catchpoint: " file=basicCatchup_test.go function=github.com/algorand/go-algorand/test/e2e-go/features/catchup.TestBasicCatchup.func1 line=93
    testingLogger.go:38: time="2023-03-13T14:24:43.002523 -0400" level=info msg="currend round: 4, primary round: 4, current-sync time and time since last round 4.858446, 0.856128, prim-sync time and time since last round: 0.000000, 1.623189, 2nd node catchpoint: " file=basicCatchup_test.go function=github.com/algorand/go-algorand/test/e2e-go/features/catchup.TestBasicCatchup.func1 line=93
    testingLogger.go:38: time="2023-03-13T14:24:43.253797 -0400" level=info msg="currend round: 4, primary round: 4, current-sync time and time since last round 5.109628, 1.107307, prim-sync time and time since last round: 0.000000, 1.874394, 2nd node catchpoint: " file=basicCatchup_test.go function=github.com/algorand/go-algorand/test/e2e-go/features/catchup.TestBasicCatchup.func1 line=93
    testingLogger.go:38: time="2023-03-13T14:24:43.505045 -0400" level=info msg="currend round: 4, primary round: 4, current-sync time and time since last round 5.360840, 1.358516, prim-sync time and time since last round: 0.000000, 2.125644, 2nd node catchpoint: " file=basicCatchup_test.go function=github.com/algorand/go-algorand/test/e2e-go/features/catchup.TestBasicCatchup.func1 line=93
    testingLogger.go:38: time="2023-03-13T14:24:43.757145 -0400" level=info msg="currend round: 4, primary round: 4, current-sync time and time since last round 5.613037, 1.610710, prim-sync time and time since last round: 0.000000, 2.377811, 2nd node catchpoint: " file=basicCatchup_test.go function=github.com/algorand/go-algorand/test/e2e-go/features/catchup.TestBasicCatchup.func1 line=93
    testingLogger.go:38: time="2023-03-13T14:24:44.008320 -0400" level=info msg="currend round: 4, primary round: 4, current-sync time and time since last round 5.864140, 1.861811, prim-sync time and time since last round: 0.000000, 2.628952, 2nd node catchpoint: " file=basicCatchup_test.go function=github.com/algorand/go-algorand/test/e2e-go/features/catchup.TestBasicCatchup.func1 line=93
    testingLogger.go:38: time="2023-03-13T14:24:44.260556 -0400" level=info msg="currend round: 4, primary round: 4, current-sync time and time since last round 6.116262, 2.113930, prim-sync time and time since last round: 0.000000, 2.881087, 2nd node catchpoint: " file=basicCatchup_test.go function=github.com/algorand/go-algorand/test/e2e-go/features/catchup.TestBasicCatchup.func1 line=93
    testingLogger.go:38: time="2023-03-13T14:24:44.511582 -0400" level=info msg="currend round: 4, primary round: 4, current-sync time and time since last round 6.367488, 2.365154, prim-sync time and time since last round: 0.000000, 3.132220, 2nd node catchpoint: " file=basicCatchup_test.go function=github.com/algorand/go-algorand/test/e2e-go/features/catchup.TestBasicCatchup.func1 line=93
    testingLogger.go:38: time="2023-03-13T14:24:44.763226 -0400" level=info msg="currend round: 4, primary round: 4, current-sync time and time since last round 6.618728, 2.616391, prim-sync time and time since last round: 0.000000, 3.383849, 2nd node catchpoint: " file=basicCatchup_test.go function=github.com/algorand/go-algorand/test/e2e-go/features/catchup.TestBasicCatchup.func1 line=93
    testingLogger.go:38: time="2023-03-13T14:24:45.014588 -0400" level=info msg="currend round: 5, primary round: 5, current-sync time and time since last round 6.870412, 0.193090, prim-sync time and time since last round: 0.000000, 0.215172, 2nd node catchpoint: " file=basicCatchup_test.go function=github.com/algorand/go-algorand/test/e2e-go/features/catchup.TestBasicCatchup.func1 line=93
    testingLogger.go:38: time="2023-03-13T14:24:45.265631 -0400" level=info msg="currend round: 5, primary round: 5, current-sync time and time since last round 7.121452, 0.444127, prim-sync time and time since last round: 0.000000, 0.466224, 2nd node catchpoint: " file=basicCatchup_test.go function=github.com/algorand/go-algorand/test/e2e-go/features/catchup.TestBasicCatchup.func1 line=93
    testingLogger.go:38: time="2023-03-13T14:24:45.517110 -0400" level=info msg="currend round: 5, primary round: 5, current-sync time and time since last round 7.372796, 0.695469, prim-sync time and time since last round: 0.000000, 0.717715, 2nd node catchpoint: " file=basicCatchup_test.go function=github.com/algorand/go-algorand/test/e2e-go/features/catchup.TestBasicCatchup.func1 line=93
    testingLogger.go:38: time="2023-03-13T14:24:45.768740 -0400" level=info msg="currend round: 5, primary round: 5, current-sync time and time since last round 7.624620, 0.947290, prim-sync time and time since last round: 0.000000, 0.969344, 2nd node catchpoint: " file=basicCatchup_test.go function=github.com/algorand/go-algorand/test/e2e-go/features/catchup.TestBasicCatchup.func1 line=93
    testingLogger.go:38: time="2023-03-13T14:24:46.020709 -0400" level=info msg="currend round: 5, primary round: 5, current-sync time and time since last round 7.876574, 1.199241, prim-sync time and time since last round: 0.000000, 1.221289, 2nd node catchpoint: " file=basicCatchup_test.go function=github.com/algorand/go-algorand/test/e2e-go/features/catchup.TestBasicCatchup.func1 line=93
    testingLogger.go:38: time="2023-03-13T14:24:46.271644 -0400" level=info msg="currend round: 5, primary round: 5, current-sync time and time since last round 8.127557, 1.450222, prim-sync time and time since last round: 0.000000, 1.472272, 2nd node catchpoint: " file=basicCatchup_test.go function=github.com/algorand/go-algorand/test/e2e-go/features/catchup.TestBasicCatchup.func1 line=93
    testingLogger.go:38: time="2023-03-13T14:24:46.523731 -0400" level=info msg="currend round: 5, primary round: 5, current-sync time and time since last round 8.379563, 1.702225, prim-sync time and time since last round: 0.000000, 1.724303, 2nd node catchpoint: " file=basicCatchup_test.go function=github.com/algorand/go-algorand/test/e2e-go/features/catchup.TestBasicCatchup.func1 line=93
    testingLogger.go:38: time="2023-03-13T14:24:46.775645 -0400" level=info msg="currend round: 5, primary round: 5, current-sync time and time since last round 8.631577, 1.954236, prim-sync time and time since last round: 0.000000, 1.976280, 2nd node catchpoint: " file=basicCatchup_test.go function=github.com/algorand/go-algorand/test/e2e-go/features/catchup.TestBasicCatchup.func1 line=93
    testingLogger.go:38: time="2023-03-13T14:24:47.026607 -0400" level=info msg="currend round: 5, primary round: 5, current-sync time and time since last round 8.882535, 2.205191, prim-sync time and time since last round: 0.000000, 2.227214, 2nd node catchpoint: " file=basicCatchup_test.go function=github.com/algorand/go-algorand/test/e2e-go/features/catchup.TestBasicCatchup.func1 line=93
    testingLogger.go:38: time="2023-03-13T14:24:47.278222 -0400" level=info msg="currend round: 5, primary round: 5, current-sync time and time since last round 0.000000, 2.456737, prim-sync time and time since last round: 0.000000, 2.478810, 2nd node catchpoint: " file=basicCatchup_test.go function=github.com/algorand/go-algorand/test/e2e-go/features/catchup.TestBasicCatchup.func1 line=93
    testingLogger.go:38: time="2023-03-13T14:24:47.530244 -0400" level=info msg="currend round: 5, primary round: 5, current-sync time and time since last round 0.000000, 2.708791, prim-sync time and time since last round: 0.000000, 2.730856, 2nd node catchpoint: " file=basicCatchup_test.go function=github.com/algorand/go-algorand/test/e2e-go/features/catchup.TestBasicCatchup.func1 line=93
    testingLogger.go:38: time="2023-03-13T14:24:47.781638 -0400" level=info msg="currend round: 5, primary round: 5, current-sync time and time since last round 0.000000, 2.960197, prim-sync time and time since last round: 0.000000, 2.982246, 2nd node catchpoint: " file=basicCatchup_test.go function=github.com/algorand/go-algorand/test/e2e-go/features/catchup.TestBasicCatchup.func1 line=93
    testingLogger.go:38: time="2023-03-13T14:24:48.033135 -0400" level=info msg="currend round: 5, primary round: 5, current-sync time and time since last round 0.000000, 3.211590, prim-sync time and time since last round: 0.000000, 3.233655, 2nd node catchpoint: " file=basicCatchup_test.go function=github.com/algorand/go-algorand/test/e2e-go/features/catchup.TestBasicCatchup.func1 line=93
    testingLogger.go:38: time="2023-03-13T14:24:48.284726 -0400" level=info msg="currend round: 6, primary round: 6, current-sync time and time since last round 0.000000, 0.062475, prim-sync time and time since last round: 0.000000, 0.072758, 2nd node catchpoint: " file=basicCatchup_test.go function=github.com/algorand/go-algorand/test/e2e-go/features/catchup.TestBasicCatchup.func1 line=93
    testingLogger.go:38: time="2023-03-13T14:24:48.536103 -0400" level=info msg="currend round: 6, primary round: 6, current-sync time and time since last round 0.000000, 0.313806, prim-sync time and time since last round: 0.000000, 0.324141, 2nd node catchpoint: " file=basicCatchup_test.go function=github.com/algorand/go-algorand/test/e2e-go/features/catchup.TestBasicCatchup.func1 line=93
    testingLogger.go:38: time="2023-03-13T14:24:48.787890 -0400" level=info msg="currend round: 6, primary round: 6, current-sync time and time since last round 0.000000, 0.565640, prim-sync time and time since last round: 0.000000, 0.575930, 2nd node catchpoint: " file=basicCatchup_test.go function=github.com/algorand/go-algorand/test/e2e-go/features/catchup.TestBasicCatchup.func1 line=93
    testingLogger.go:38: time="2023-03-13T14:24:49.039906 -0400" level=info msg="currend round: 6, primary round: 6, current-sync time and time since last round 0.000000, 0.817661, prim-sync time and time since last round: 0.000000, 0.827946, 2nd node catchpoint: " file=basicCatchup_test.go function=github.com/algorand/go-algorand/test/e2e-go/features/catchup.TestBasicCatchup.func1 line=93
    testingLogger.go:38: time="2023-03-13T14:24:49.291890 -0400" level=info msg="currend round: 6, primary round: 6, current-sync time and time since last round 0.000000, 1.069653, prim-sync time and time since last round: 0.000000, 1.079940, 2nd node catchpoint: " file=basicCatchup_test.go function=github.com/algorand/go-algorand/test/e2e-go/features/catchup.TestBasicCatchup.func1 line=93
    testingLogger.go:38: time="2023-03-13T14:24:49.543729 -0400" level=info msg="currend round: 6, primary round: 6, current-sync time and time since last round 0.000000, 1.321466, prim-sync time and time since last round: 0.000000, 1.331708, 2nd node catchpoint: " file=basicCatchup_test.go function=github.com/algorand/go-algorand/test/e2e-go/features/catchup.TestBasicCatchup.func1 line=93
    testingLogger.go:38: time="2023-03-13T14:24:49.795723 -0400" level=info msg="currend round: 6, primary round: 6, current-sync time and time since last round 0.000000, 1.573460, prim-sync time and time since last round: 0.000000, 1.583754, 2nd node catchpoint: " file=basicCatchup_test.go function=github.com/algorand/go-algorand/test/e2e-go/features/catchup.TestBasicCatchup.func1 line=93
    testingLogger.go:38: time="2023-03-13T14:24:50.047055 -0400" level=info msg="currend round: 6, primary round: 6, current-sync time and time since last round 0.000000, 1.824845, prim-sync time and time since last round: 0.000000, 1.835093, 2nd node catchpoint: " file=basicCatchup_test.go function=github.com/algorand/go-algorand/test/e2e-go/features/catchup.TestBasicCatchup.func1 line=93
    testingLogger.go:38: time="2023-03-13T14:24:50.298084 -0400" level=info msg="currend round: 6, primary round: 6, current-sync time and time since last round 0.000000, 2.075869, prim-sync time and time since last round: 0.000000, 2.086131, 2nd node catchpoint: " file=basicCatchup_test.go function=github.com/algorand/go-algorand/test/e2e-go/features/catchup.TestBasicCatchup.func1 line=93
    testingLogger.go:38: time="2023-03-13T14:24:50.549667 -0400" level=info msg="currend round: 6, primary round: 6, current-sync time and time since last round 0.000000, 2.327428, prim-sync time and time since last round: 0.000000, 2.337672, 2nd node catchpoint: " file=basicCatchup_test.go function=github.com/algorand/go-algorand/test/e2e-go/features/catchup.TestBasicCatchup.func1 line=93
    testingLogger.go:38: time="2023-03-13T14:24:50.800691 -0400" level=info msg="currend round: 6, primary round: 6, current-sync time and time since last round 0.000000, 2.578462, prim-sync time and time since last round: 0.000000, 2.588731, 2nd node catchpoint: " file=basicCatchup_test.go function=github.com/algorand/go-algorand/test/e2e-go/features/catchup.TestBasicCatchup.func1 line=93
    testingLogger.go:38: time="2023-03-13T14:24:51.052056 -0400" level=info msg="currend round: 6, primary round: 6, current-sync time and time since last round 0.000000, 2.829477, prim-sync time and time since last round: 0.000000, 2.839920, 2nd node catchpoint: " file=basicCatchup_test.go function=github.com/algorand/go-algorand/test/e2e-go/features/catchup.TestBasicCatchup.func1 line=93
    testingLogger.go:38: time="2023-03-13T14:24:51.304958 -0400" level=info msg="currend round: 6, primary round: 6, current-sync time and time since last round 0.000000, 3.082613, prim-sync time and time since last round: 0.000000, 3.092960, 2nd node catchpoint: " file=basicCatchup_test.go function=github.com/algorand/go-algorand/test/e2e-go/features/catchup.TestBasicCatchup.func1 line=93
    testingLogger.go:38: time="2023-03-13T14:24:51.556517 -0400" level=info msg="currend round: 6, primary round: 6, current-sync time and time since last round 0.000000, 3.334290, prim-sync time and time since last round: 0.000000, 3.344542, 2nd node catchpoint: " file=basicCatchup_test.go function=github.com/algorand/go-algorand/test/e2e-go/features/catchup.TestBasicCatchup.func1 line=93
    testingLogger.go:38: time="2023-03-13T14:24:51.808300 -0400" level=info msg="currend round: 7, primary round: 7, current-sync time and time since last round 0.000000, 0.160722, prim-sync time and time since last round: 0.000000, 0.180776, 2nd node catchpoint: " file=basicCatchup_test.go function=github.com/algorand/go-algorand/test/e2e-go/features/catchup.TestBasicCatchup.func1 line=93
    testingLogger.go:38: time="2023-03-13T14:24:52.059961 -0400" level=info msg="currend round: 7, primary round: 7, current-sync time and time since last round 0.000000, 0.412355, prim-sync time and time since last round: 0.000000, 0.432434, 2nd node catchpoint: " file=basicCatchup_test.go function=github.com/algorand/go-algorand/test/e2e-go/features/catchup.TestBasicCatchup.func1 line=93
    testingLogger.go:38: time="2023-03-13T14:24:52.311914 -0400" level=info msg="currend round: 7, primary round: 7, current-sync time and time since last round 0.000000, 0.664425, prim-sync time and time since last round: 0.000000, 0.684437, 2nd node catchpoint: " file=basicCatchup_test.go function=github.com/algorand/go-algorand/test/e2e-go/features/catchup.TestBasicCatchup.func1 line=93
    testingLogger.go:38: time="2023-03-13T14:24:52.563741 -0400" level=info msg="currend round: 7, primary round: 7, current-sync time and time since last round 0.000000, 0.916191, prim-sync time and time since last round: 0.000000, 0.936265, 2nd node catchpoint: " file=basicCatchup_test.go function=github.com/algorand/go-algorand/test/e2e-go/features/catchup.TestBasicCatchup.func1 line=93
    testingLogger.go:38: time="2023-03-13T14:24:52.815729 -0400" level=info msg="currend round: 7, primary round: 7, current-sync time and time since last round 0.000000, 1.168191, prim-sync time and time since last round: 0.000000, 1.188197, 2nd node catchpoint: " file=basicCatchup_test.go function=github.com/algorand/go-algorand/test/e2e-go/features/catchup.TestBasicCatchup.func1 line=93
    testingLogger.go:38: time="2023-03-13T14:24:53.067027 -0400" level=info msg="currend round: 7, primary round: 7, current-sync time and time since last round 0.000000, 1.419456, prim-sync time and time since last round: 0.000000, 1.439517, 2nd node catchpoint: " file=basicCatchup_test.go function=github.com/algorand/go-algorand/test/e2e-go/features/catchup.TestBasicCatchup.func1 line=93
    testingLogger.go:38: time="2023-03-13T14:24:53.318873 -0400" level=info msg="currend round: 7, primary round: 7, current-sync time and time since last round 0.000000, 1.671304, prim-sync time and time since last round: 0.000000, 1.691363, 2nd node catchpoint: " file=basicCatchup_test.go function=github.com/algorand/go-algorand/test/e2e-go/features/catchup.TestBasicCatchup.func1 line=93
    testingLogger.go:38: time="2023-03-13T14:24:53.570878 -0400" level=info msg="currend round: 7, primary round: 7, current-sync time and time since last round 0.000000, 1.923308, prim-sync time and time since last round: 0.000000, 1.943365, 2nd node catchpoint: " file=basicCatchup_test.go function=github.com/algorand/go-algorand/test/e2e-go/features/catchup.TestBasicCatchup.func1 line=93
    testingLogger.go:38: time="2023-03-13T14:24:53.822634 -0400" level=info msg="currend round: 7, primary round: 7, current-sync time and time since last round 0.000000, 2.175107, prim-sync time and time since last round: 0.000000, 2.195129, 2nd node catchpoint: " file=basicCatchup_test.go function=github.com/algorand/go-algorand/test/e2e-go/features/catchup.TestBasicCatchup.func1 line=93
    testingLogger.go:38: time="2023-03-13T14:24:54.074614 -0400" level=info msg="currend round: 7, primary round: 7, current-sync time and time since last round 0.000000, 2.427059, prim-sync time and time since last round: 0.000000, 2.447090, 2nd node catchpoint: " file=basicCatchup_test.go function=github.com/algorand/go-algorand/test/e2e-go/features/catchup.TestBasicCatchup.func1 line=93
    testingLogger.go:38: time="2023-03-13T14:24:54.325600 -0400" level=info msg="currend round: 7, primary round: 7, current-sync time and time since last round 0.000000, 2.678063, prim-sync time and time since last round: 0.000000, 2.698087, 2nd node catchpoint: " file=basicCatchup_test.go function=github.com/algorand/go-algorand/test/e2e-go/features/catchup.TestBasicCatchup.func1 line=93
    testingLogger.go:38: time="2023-03-13T14:24:54.577619 -0400" level=info msg="currend round: 7, primary round: 7, current-sync time and time since last round 0.000000, 2.930033, prim-sync time and time since last round: 0.000000, 2.950106, 2nd node catchpoint: " file=basicCatchup_test.go function=github.com/algorand/go-algorand/test/e2e-go/features/catchup.TestBasicCatchup.func1 line=93
    testingLogger.go:38: time="2023-03-13T14:24:54.829729 -0400" level=info msg="currend round: 7, primary round: 7, current-sync time and time since last round 0.000000, 3.182097, prim-sync time and time since last round: 0.000000, 3.202161, 2nd node catchpoint: " file=basicCatchup_test.go function=github.com/algorand/go-algorand/test/e2e-go/features/catchup.TestBasicCatchup.func1 line=93
    testingLogger.go:38: time="2023-03-13T14:24:55.083271 -0400" level=info msg="currend round: 8, primary round: 8, current-sync time and time since last round 0.000000, 0.018172, prim-sync time and time since last round: 0.000000, 0.037080, 2nd node catchpoint: " file=basicCatchup_test.go function=github.com/algorand/go-algorand/test/e2e-go/features/catchup.TestBasicCatchup.func1 line=93
    testingLogger.go:38: time="2023-03-13T14:24:55.338995 -0400" level=info msg="currend round: 8, primary round: 8, current-sync time and time since last round 0.000000, 0.275079, prim-sync time and time since last round: 0.000000, 0.292837, 2nd node catchpoint: " file=basicCatchup_test.go function=github.com/algorand/go-algorand/test/e2e-go/features/catchup.TestBasicCatchup.func1 line=93
    testingLogger.go:38: time="2023-03-13T14:24:55.590212 -0400" level=info msg="currend round: 8, primary round: 8, current-sync time and time since last round 0.000000, 0.526239, prim-sync time and time since last round: 0.000000, 0.544072, 2nd node catchpoint: " file=basicCatchup_test.go function=github.com/algorand/go-algorand/test/e2e-go/features/catchup.TestBasicCatchup.func1 line=93
    testingLogger.go:38: time="2023-03-13T14:24:55.842147 -0400" level=info msg="currend round: 8, primary round: 8, current-sync time and time since last round 0.000000, 0.778298, prim-sync time and time since last round: 0.000000, 0.796005, 2nd node catchpoint: " file=basicCatchup_test.go function=github.com/algorand/go-algorand/test/e2e-go/features/catchup.TestBasicCatchup.func1 line=93
    testingLogger.go:38: time="2023-03-13T14:24:56.094741 -0400" level=info msg="currend round: 8, primary round: 8, current-sync time and time since last round 0.000000, 1.030871, prim-sync time and time since last round: 0.000000, 1.048598, 2nd node catchpoint: " file=basicCatchup_test.go function=github.com/algorand/go-algorand/test/e2e-go/features/catchup.TestBasicCatchup.func1 line=93
    testingLogger.go:38: time="2023-03-13T14:24:56.345894 -0400" level=info msg="currend round: 8, primary round: 8, current-sync time and time since last round 0.000000, 1.282087, prim-sync time and time since last round: 0.000000, 1.299777, 2nd node catchpoint: " file=basicCatchup_test.go function=github.com/algorand/go-algorand/test/e2e-go/features/catchup.TestBasicCatchup.func1 line=93
    testingLogger.go:38: time="2023-03-13T14:24:56.597922 -0400" level=info msg="currend round: 8, primary round: 8, current-sync time and time since last round 0.000000, 1.534057, prim-sync time and time since last round: 0.000000, 1.551751, 2nd node catchpoint: " file=basicCatchup_test.go function=github.com/algorand/go-algorand/test/e2e-go/features/catchup.TestBasicCatchup.func1 line=93
    testingLogger.go:38: time="2023-03-13T14:24:56.849839 -0400" level=info msg="currend round: 8, primary round: 8, current-sync time and time since last round 0.000000, 1.785987, prim-sync time and time since last round: 0.000000, 1.803700, 2nd node catchpoint: " file=basicCatchup_test.go function=github.com/algorand/go-algorand/test/e2e-go/features/catchup.TestBasicCatchup.func1 line=93
    testingLogger.go:38: time="2023-03-13T14:24:57.101806 -0400" level=info msg="currend round: 8, primary round: 8, current-sync time and time since last round 0.000000, 2.037986, prim-sync time and time since last round: 0.000000, 2.055681, 2nd node catchpoint: " file=basicCatchup_test.go function=github.com/algorand/go-algorand/test/e2e-go/features/catchup.TestBasicCatchup.func1 line=93
    testingLogger.go:38: time="2023-03-13T14:24:57.353111 -0400" level=info msg="currend round: 8, primary round: 8, current-sync time and time since last round 0.000000, 2.289254, prim-sync time and time since last round: 0.000000, 2.306944, 2nd node catchpoint: " file=basicCatchup_test.go function=github.com/algorand/go-algorand/test/e2e-go/features/catchup.TestBasicCatchup.func1 line=93
    testingLogger.go:38: time="2023-03-13T14:24:57.604638 -0400" level=info msg="currend round: 8, primary round: 8, current-sync time and time since last round 0.000000, 2.540789, prim-sync time and time since last round: 0.000000, 2.558507, 2nd node catchpoint: " file=basicCatchup_test.go function=github.com/algorand/go-algorand/test/e2e-go/features/catchup.TestBasicCatchup.func1 line=93
    testingLogger.go:38: time="2023-03-13T14:24:57.856700 -0400" level=info msg="currend round: 8, primary round: 8, current-sync time and time since last round 0.000000, 2.792770, prim-sync time and time since last round: 0.000000, 2.810471, 2nd node catchpoint: " file=basicCatchup_test.go function=github.com/algorand/go-algorand/test/e2e-go/features/catchup.TestBasicCatchup.func1 line=93
    testingLogger.go:38: time="2023-03-13T14:24:58.108856 -0400" level=info msg="currend round: 8, primary round: 8, current-sync time and time since last round 0.000000, 3.044907, prim-sync time and time since last round: 0.000000, 3.062696, 2nd node catchpoint: " file=basicCatchup_test.go function=github.com/algorand/go-algorand/test/e2e-go/features/catchup.TestBasicCatchup.func1 line=93
    testingLogger.go:38: time="2023-03-13T14:24:58.360768 -0400" level=info msg="currend round: 8, primary round: 8, current-sync time and time since last round 0.000000, 3.296749, prim-sync time and time since last round: 0.000000, 3.314586, 2nd node catchpoint: " file=basicCatchup_test.go function=github.com/algorand/go-algorand/test/e2e-go/features/catchup.TestBasicCatchup.func1 line=93
    testingLogger.go:38: time="2023-03-13T14:24:58.612720 -0400" level=info msg="currend round: 9, primary round: 9, current-sync time and time since last round 0.000000, 0.136541, prim-sync time and time since last round: 0.000000, 0.151237, 2nd node catchpoint: " file=basicCatchup_test.go function=github.com/algorand/go-algorand/test/e2e-go/features/catchup.TestBasicCatchup.func1 line=93
    testingLogger.go:38: time="2023-03-13T14:24:58.864205 -0400" level=info msg="currend round: 9, primary round: 9, current-sync time and time since last round 0.000000, 0.387967, prim-sync time and time since last round: 0.000000, 0.402749, 2nd node catchpoint: " file=basicCatchup_test.go function=github.com/algorand/go-algorand/test/e2e-go/features/catchup.TestBasicCatchup.func1 line=93
    testingLogger.go:38: time="2023-03-13T14:24:59.115365 -0400" level=info msg="currend round: 9, primary round: 9, current-sync time and time since last round 0.000000, 0.639098, prim-sync time and time since last round: 0.000000, 0.653875, 2nd node catchpoint: " file=basicCatchup_test.go function=github.com/algorand/go-algorand/test/e2e-go/features/catchup.TestBasicCatchup.func1 line=93
    testingLogger.go:38: time="2023-03-13T14:24:59.366725 -0400" level=info msg="currend round: 9, primary round: 9, current-sync time and time since last round 0.000000, 0.890472, prim-sync time and time since last round: 0.000000, 0.905213, 2nd node catchpoint: " file=basicCatchup_test.go function=github.com/algorand/go-algorand/test/e2e-go/features/catchup.TestBasicCatchup.func1 line=93
    testingLogger.go:38: time="2023-03-13T14:24:59.618653 -0400" level=info msg="currend round: 9, primary round: 9, current-sync time and time since last round 0.000000, 1.142381, prim-sync time and time since last round: 0.000000, 1.157134, 2nd node catchpoint: " file=basicCatchup_test.go function=github.com/algorand/go-algorand/test/e2e-go/features/catchup.TestBasicCatchup.func1 line=93
    testingLogger.go:38: time="2023-03-13T14:24:59.870661 -0400" level=info msg="currend round: 9, primary round: 9, current-sync time and time since last round 0.000000, 1.394443, prim-sync time and time since last round: 0.000000, 1.409160, 2nd node catchpoint: " file=basicCatchup_test.go function=github.com/algorand/go-algorand/test/e2e-go/features/catchup.TestBasicCatchup.func1 line=93
    testingLogger.go:38: time="2023-03-13T14:25:00.122480 -0400" level=info msg="currend round: 9, primary round: 9, current-sync time and time since last round 0.000000, 1.645928, prim-sync time and time since last round: 0.000000, 1.661011, 2nd node catchpoint: " file=basicCatchup_test.go function=github.com/algorand/go-algorand/test/e2e-go/features/catchup.TestBasicCatchup.func1 line=93
    testingLogger.go:38: time="2023-03-13T14:25:00.374379 -0400" level=info msg="currend round: 9, primary round: 9, current-sync time and time since last round 0.000000, 1.898127, prim-sync time and time since last round: 0.000000, 1.912863, 2nd node catchpoint: " file=basicCatchup_test.go function=github.com/algorand/go-algorand/test/e2e-go/features/catchup.TestBasicCatchup.func1 line=93
    testingLogger.go:38: time="2023-03-13T14:25:00.625879 -0400" level=info msg="currend round: 9, primary round: 9, current-sync time and time since last round 0.000000, 2.149593, prim-sync time and time since last round: 0.000000, 2.164334, 2nd node catchpoint: " file=basicCatchup_test.go function=github.com/algorand/go-algorand/test/e2e-go/features/catchup.TestBasicCatchup.func1 line=93
    testingLogger.go:38: time="2023-03-13T14:25:00.877306 -0400" level=info msg="currend round: 9, primary round: 9, current-sync time and time since last round 0.000000, 2.401086, prim-sync time and time since last round: 0.000000, 2.415794, 2nd node catchpoint: " file=basicCatchup_test.go function=github.com/algorand/go-algorand/test/e2e-go/features/catchup.TestBasicCatchup.func1 line=93
    testingLogger.go:38: time="2023-03-13T14:25:01.128445 -0400" level=info msg="currend round: 9, primary round: 9, current-sync time and time since last round 0.000000, 2.652207, prim-sync time and time since last round: 0.000000, 2.666954, 2nd node catchpoint: " file=basicCatchup_test.go function=github.com/algorand/go-algorand/test/e2e-go/features/catchup.TestBasicCatchup.func1 line=93
    testingLogger.go:38: time="2023-03-13T14:25:01.380152 -0400" level=info msg="currend round: 9, primary round: 9, current-sync time and time since last round 0.000000, 2.903958, prim-sync time and time since last round: 0.000000, 2.918659, 2nd node catchpoint: " file=basicCatchup_test.go function=github.com/algorand/go-algorand/test/e2e-go/features/catchup.TestBasicCatchup.func1 line=93
    testingLogger.go:38: time="2023-03-13T14:25:01.631303 -0400" level=info msg="currend round: 9, primary round: 9, current-sync time and time since last round 0.000000, 3.155004, prim-sync time and time since last round: 0.000000, 3.169770, 2nd node catchpoint: " file=basicCatchup_test.go function=github.com/algorand/go-algorand/test/e2e-go/features/catchup.TestBasicCatchup.func1 line=93
    testingLogger.go:38: time="2023-03-13T14:25:01.896044 -0400" level=info msg="currend round: 9, primary round: 10, current-sync time and time since last round 0.000000, 3.406013, prim-sync time and time since last round: 0.000000, 0.013264, 2nd node catchpoint: " file=basicCatchup_test.go function=github.com/algorand/go-algorand/test/e2e-go/features/catchup.TestBasicCatchup.func1 line=93
    testingLogger.go:38: time="2023-03-13T14:25:02.157139 -0400" level=info msg="currend round: 10, primary round: 10, current-sync time and time since last round 0.000000, 0.260509, prim-sync time and time since last round: 0.000000, 0.275033, 2nd node catchpoint: " file=basicCatchup_test.go function=github.com/algorand/go-algorand/test/e2e-go/features/catchup.TestBasicCatchup.func1 line=93
```